deployment/lambdas/submit_job/lambda_entrypoint.py

```diff
@@ -89,6 +93,27 @@ def main(event, context):
     data = get_submission_data(tumor_sample_md, normal_sample_md, subject_id, api_auth)
     LOGGER.debug(f'compiled submission data: {data}')
 
+    # Abort job submission, if job_name is in gpl-job-queue
+    job_list = CLIENT_BATCH.list_jobs(
+        jobQueue=BATCH_QUEUE_NAME,
+        filters=[
+            {
+                'name': 'AFTER_CREATED_AT',
+                'values': [
+                    '0',
+                ]
+            },
+        ]
+    )
+    for job in job_list['jobSummaryList']:
+        existing_job_name = job['jobName']
+        if data['job_name'] in existing_job_name:
+            # no-ops
+            return {
+                'statusCode': 202,
+                'body': f'Subject {subject_id} has existing batch job with name {existing_job_name}',
+            }
```

Comment:
The `Batch.Client.list_jobs` call returns a dictionary containing job summaries under the `jobSummaryList` key - should we use `==` inplace of `in` here as below?

```python
for job in job_list['jobSummaryList']:
    if data['job_name'] == job['jobName']:
        return {
            'statusCode': 202,
            'body': f'Subject {subject_id} has existing batch job with name {existing_job_name}',
        }
```

Reply:
Ok, will do.

Reply:
Fixed with https://github.com/umccr/gridss-purple-linx-nf/commit/c7c52d5c191bfb86352004e9a404739ebd9205c8